docs/guides/deploy-guide/manager.md

```diff
@@ -112,16 +123,18 @@ in the Ansible documentation.
 | `ANSIBLE_BECOME_ASK_PASS` | Boolean | Toggle to prompt for privilege escalation password.                                                                                                                           |
 | `ANSIBLE_SSH_ARGS`        | String  | If set, this will override the Ansible default ssh arguments.                                                                                                                 |
 | `ANSIBLE_USER`            | String  | The user Ansible ‘logs in’ as.                                                                                                                                                |
-To verify the creation of the operator user, use the private key file `id_rsa.operator`. Make
+
+
+To verify the proper creation of the operator user, use the private key file `id_rsa.operator`. Make
 sure you purge all keys from ssh-agent identity cache using `ssh-add -D`. You can print the list
 using `ssh-add -l`. The list should be empty.
 
 ```
 ssh-add -D
-ssh -o IdentitiesOnly=yes -i id_rsa.operator dragon@testbed-manager
+ssh -o IdentitiesOnly=yes -i  ../../secrets/id_rsa.operator dragon@YOUR_MANAGER_NODE
```

Comment:
At the point the id_rsa.operator file is available in environments/manager and should be used. We cannot assume that the secrets directory is available on the seed node.

Reply:
Added a placeholder instead